deployment/terraform/modules/docker_host/main.tf

```diff
@@ -0,0 +1,57 @@
+variable "cidr_block" {
+  type = "string"
+  default = "10.0.0.0/16"
```

Comment:
It looks like both Orchestra and Odyssey are using 10.0.0.0/8, so I'd change this to either 172.16.0.0/16 or 192.168.0.0/16 to prevent any potential conflicts if there is ever a need for peering.
https://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/VPC_Subnets.html#VPC_Sizing